src/lib.rs

```diff
@@ -407,7 +419,17 @@ impl Plugin for CosmicEditPlugin {
                 swash_cache: SwashCache::new(),
             })
             .insert_resource(CosmicFontSystem(font_system))
+            // Cursor Bits
             .add_event::<CosmicTextChanged>();
+
+        // Cursor Bits
+        app.add_systems(Update, (hover_sprites, hover_ui))
```

Comment:
Let's hide it under the `change_cursor` flag too.

Reply:
For custom cursors I would prefer config options for cursors.... but `change_cursor` with default | events | none also should be fine 👍 I just want to have an option to fully disable it if needed.

Reply:
Custom cursor configs would be simpler if bevy_cosmic_edit was the only one controlling the cursor, but I feel if there are other plugins or code that also need cursor control hooking an event would be easier to integrate. Full disable a good shout, no need for unused events